UnityProject/Assets/Scripts/Items/Weapons/ElectricalMagazine.cs

```diff
@@ -40,4 +43,15 @@ public override String Examine(Vector3 pos)
 		double percent = (battery.Watts * 100 / battery.MaxWatts);
 		return $"It seems to be compatible with energy weapons. The charge indicator displays {Math.Round(percent)} percent.";
 	}
+
+	#if UNITY_EDITOR
+	[CustomEditor(typeof(MagazineBehaviour), true)]
+	public class MagEditor : Editor
+	{
+		public override void OnInspectorGUI()
+		{
+			DrawDefaultInspector();
```

Comment:
I think this is not needed if all we're doing is drawing the normal inspector view

Reply:
as far as im aware this is the only method of overriding the inspector scripts attached to magazinebehaviour.cs

Reply:
oh, I see. The super class modified the inspector and these child classes need the normal inspector back.